src/LogicalGraph.ts

```diff
@@ -945,10 +945,78 @@ export class LogicalGraph {
+        if (graph.activeGraphConfigId() !== undefined && graph.activeGraphConfigId() !== ""){
+            if (graph.getActiveGraphConfig() === null){
+                const issue: Errors.Issue = Errors.Fix(
+                    "Active Graph Config Id (" + graph.activeGraphConfigId() + ") does not match a known graph config",
+                    function(){
+                        // if there are no graph config, set active id to undefined
+                        // otherwise, just set the active id to the id of the first graph config in the list
+                        if (graph.getGraphConfigs().length === 0){
+                            graph.setActiveGraphConfig(undefined);
+                        } else {
+                            graph.setActiveGraphConfig(graph.getGraphConfigs()[0].getId());
+                        }
+                    },
+                    "Make first graph config active, or set undefined if no graph configs present"
+                );
+                graph.issues.push({issue : issue, validity : Errors.Validity.Error})
+            }
+        }
+
+        // check that all fields, in all nodes, in all graph configs are actually present in the graph
+        for (const graphConfig of graph.getGraphConfigs()){
+            for (const graphConfigNode of graphConfig.getNodes()){
+                // check that node exists in graph
+                const graphNode: Node = graph.findNodeByIdQuiet(graphConfigNode.getId());
+
+                if (graphNode === null){
+                    const issue: Errors.Issue = Errors.Fix(
+                        "Node (" + graphConfigNode.getId() +") in graph config (" + graphConfig.getName() + ") is not present in Logical Graph",
+                        function(){
+                            graphConfig.removeNode(graphConfigNode);
+                        },
+                        "Delete node from graph config"
+                    );
+                    graph.issues.push({issue : issue, validity : Errors.Validity.Error});
+                    break;
+                }
+
+                for (const graphConfigField of graphConfigNode.getFields()){
+                    const graphField: Field = graphNode.findFieldById(graphConfigField.getId());
+
+                    if (graphField === null){
+                        const issue: Errors.Issue = Errors.Fix(
+                            "Field (" + graphConfigField.getId() + ") in graph config (" + graphConfig.getName() + ", " + graphNode.getName() + ") is not present in Logical Graph",
+                            function(){
+                                graphConfigNode.removeFieldById(graphConfigField.getId());
+                            },
+                            "Delete field from node in graph config"
+                        );
+                        graph.issues.push({issue: issue, validity: Errors.Validity.Error});
+                    }
+                }
+            }
+        }
```

Comment:
i like the way this scenario is handled. The dev console is quite noisy about not being able to find a node though. perhaps we dont need to show those warnings?

Reply:
oh, i also just noticed that when the table is showing the missing node and field in the graph configuration attributes table, the tooltips cause an error when hovering on the node name.

Reply:
Yeah, true, there are a lot of warnings.

I'm using findNodeById**Quiet**() here, but there are a few other places that generated warnings. I've now changed the html data-bindings to use findNodeByIdQuiet() too.

And also removed two console.warn() that were explicitly called when getTableFields() could not find a node or field.

Reply:
The tooltip error was caused by passing the eagleTooltip binding undefined html.

I've updated the eagleTooltip code to avoid that error.